### PR TITLE
added Leo Peng to file for issue#6104

### LINKED ISSUE #6104: Update Communities of Practice: UI/UX (Add Leo Peng)
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep communities of practice information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_data/internal/communities/ui-ux.yml` file
- [x] Observe the existing syntax of the front matter block [^1] in the file
- [x] Find the `leadership` variable and add the following as a new entry:
```
  - name: Leo Peng
    role: Co-lead
    links:
      slack: https://hackforla.slack.com/team/UPDM52A94
      github: https://github.com/leopeng101
    picture: https://avatars.githubusercontent.com/leopeng101
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request: the UI/UX section on the Communities of Practice page [^2]
- [x] Once your pull request is merged, go to #6101 and
  - [x] Check off this issue under the _Dependency_ section
  - [x] If all the issues in the _Dependency_ section are checked off, move #6101 to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        
      </details>

### Resources/Instructions
- This issue was initiated by #6101 
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Communities of Practice page URL: https://www.hackforla.org/communities-of-practice

### PR DESCRIPTION
**Pull request by kevin-421**
Fixes #6104

### What changes did you make?
  - Added team member leo peng to the "_data/internal/communities/ui-ux.yml" file.

### Why did you make the changes (we will use this info to test)?
  - To keep project information up to date so that visitors can be informed on all recent changes.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details/>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
image

</details>

### CODE REVIEW

**Comment by njackman-2344** [comment]
Availability: Tuesday 1/23 7pm-10pm, Wednesday 5pm-10pm 1/24, Friday 5pm-10pm 1/25

ETA: Friday 1/25 EOD

**Comment by KyleA99** [comment]
ETA: 1/23/24 EOD
Availability: 1/23/24 all day

File: _data/internal/communities/ui-ux.yml [hunk 1]
```diff
@@ -42,6 +42,12 @@ links:
     url: https://app.slack.com/client/T04502KQX/C017ESHSMNG/
   - name: Github
     url: https://github.com/hackforla/UI-UX
+  - name: Leo Peng
```
Comment (KyleA99) [inline _data/internal/communities/ui-ux.yml:40]: Move from links section to leadership section (move to below Peter Gwon's section).

**Review by KyleA99 — CHANGES_REQUESTED** [review]
@kevin-421 

- Your PR is done with the correct branch.
- Your code needs to be modified slightly - see inline comment.
- These changes do actually affect the website visuals, please add a screenshot to your pull request once the code has been modified.

Feel free to reach out to me if you need any assistance.  

Great work

**Comment by LRenDO** [comment]
Hi @kevin-421! 

Great job with the "what" and the "why" as well as the branching! In addition to any changes requested by your reviewers, please also remove the unused bullet points in the original pull request comment. Though it's non-functional, we like to keep them clean. I would also change `team member` in the PR title to `Leo Peng` for clarity as we often have issues very similar to this one.

@KyleA99 mentioned that the visuals do change. These changes will occur on the [Communities of Practice page](https://www.hackforla.org/communities-of-practice). The affected pages are often included in the resource section of the issue.

If you have any questions, feel free to reach out. Thanks for taking the time to contribute and make updates!

**Review by KyleA99 — DISMISSED** [review]
PR approved.  Great work @kevin-421 

- Code is properly changed and looks great visually.  

Thank you for your contributions!

**Review by njackman-2344 — DISMISSED** [review]
Great work @kevin-421 

Like @KyleA99 said, Code and visuals look good.

File: _data/internal/communities/ui-ux.yml [hunk 2]
```diff
@@ -37,11 +37,16 @@ leadership:
...
+    links:
+      slack: https://hackforla.slack.com/team/UPDM52A94
+      github: https://github.com/leopeng101
+    picture: https://avatars.githubusercontent.com/leopeng101
 links:
   - name: Slack
     url: https://app.slack.com/client/T04502KQX/C017ESHSMNG/
   - name: Github
     url: https://github.com/hackforla/UI-UX
-
```
Comment (jphamtv) [inline _data/internal/communities/ui-ux.yml:45]: Please undo the removal of this blank line to maintain formatting consistency with the other yml files in the communities folder.

**Comment by jphamtv** [comment]
Hey @kevin-421, thanks for making the changes requested by @KyleA99 and @LRenDO.

I noticed the action items on the issue aren't checked off. Please make sure to mark them as completed. It helps track progress and keeps the team updated on what's done and what still needs attention.

Also, an edit was made that's out of scope. It's minor, but to maintain consistency with other yml files, revert the removal of the blank line above `recruiting-messages:`. Thanks!

**Comment by kevin-421** [comment]
Hello @jphamtv I see that you want me to check off actions items, but I cannot see where they are located can you point me in the right direction. And I also went ahead and deleted the blank line and pushed the code up thank you.

**Comment by jphamtv** [comment]
Hey @kevin-421 , I'm sorry if my previous feedback wasn't clear. Let me try explaining again: 

- The Action Items to be checked off are located in the issue here: #6104. 

- What I originally meant to say is to restore the blank line that used to be between the `links:` and `recruiting-message:` sections. Keeping the blank line there would make the formatting consistent with the other .yml files within the communities folder. Here's what lines 46–53 should look like:

```
links:
  - name: Slack
    url: https://app.slack.com/client/T04502KQX/C017ESHSMNG/
  - name: Github
    url: https://github.com/hackforla/UI-UX
    
recruiting-message:
---
```

Let me know if this makes sense. I'm happy to clarify further if needed. Please re-request reviews once you've made the changes. To re-request reviews, click the icon with the arrows next to the reviewer names. Thanks!

Screenshot 2024-01-29 at 14 55 23

**Comment by kevin-421** [comment]
yes it makes sense @jphamtv thank you for clarifying. I went ahead and make the changes let me know If anything else needs correcting. Thank you!

**Review by njackman-2344 — DISMISSED** [review]
I see the consistent code markup as suggested from @jphamtv . I apologize I didn't account for that earlier. 

The visuals support the change before and after. The branch commit matches. 

Great Job @kevin-421

File: _data/internal/communities/ui-ux.yml [hunk 3]
```diff
@@ -37,11 +37,17 @@ leadership:
...
       github: https://github.com/pogwon
     picture: https://avatars.githubusercontent.com/pogwon
-links:
+  - name: Leo Peng
+    role: Co-lead
+    links:
+      slack: https://hackforla.slack.com/team/UPDM52A94
+      github: https://github.com/leopeng101
+    picture: https://avatars.githubusercontent.com/leopeng101
+    links:
```
Comment (jphamtv) [inline _data/internal/communities/ui-ux.yml:46]: Remove indentation

**Comment by jphamtv** [comment]
> yes it makes sense @jphamtv thank you for clarifying. I went ahead and make the changes let me know If anything else needs correcting. Thank you!

Hey @kevin-421, you're almost there. Just remove the indentation on line 46 and you should be all set.

**Comment by kevin-421** [comment]
Hello @jphamtv correct me if I am wrong but I went ahead and looked at the rest of the code on the file. It looks like all of the indentation is the same for the "links:".

**Comment by KyleA99** [comment]
@kevin-421 I'll review again now and take a look at line 46

**Review by KyleA99 — CHANGES_REQUESTED** [review]
@kevin-421 I just looked over the current code for this PR and I believe that there is an indentation/nesting issue with "links:" on line 46.  Please note that it appears we are using an internal "links:" for each leadership profile, and also a separate "links:" under/after the leadership profiles.

If you refer back to the original code (https://github.com/hackforla/website/blob/gh-pages/_data/internal/communities/ui-ux.yml), the "links:" is not indented/nested inside of the last leadership profile.  I believe this issue is also throwing off the visuals, currently.
Screenshot 2024-01-30 at 1 13 02 PM

I'll be available all day if you have any questions.  Feel free to message me on slack, or tag me in a comment here and we can converse.

**Comment by Thinking-Panda** [comment]
Availability: Weekdays
Review ETA: 1/30/24

**Comment by kevin-421** [comment]
hello @KyleA99  I went ahead and removed the indentation, the visuals are now looking good. 
image

**Review by KyleA99 — APPROVED** [review]
Code and visuals appear properly changed now.  Approved! Great job @kevin-421

**Review by Thinking-Panda — APPROVED** [review]
@kevin-421 , Thank you for working on this issue. Merge branches are correct. Code changes give desired results. Website works well in local machine. Well done. Approved!

**Review by jphamtv — APPROVED** [review]
@KyleA99, thanks for providing detailed feedback on the indentation issue.

@kevin-421, appreciate your efforts on this pull request. All looks good now.

**Review by njackman-2344 — APPROVED** [review]
branches are mergeable. 

Code format is specified to standards. 

Thank you!

**Pull request merged**